SysTrayIcon/explainer.md

```diff
@@ -0,0 +1,149 @@
+
+### Instant Messaging apps
+
+![The Teams app on Windows system status icon menu](teamsMenu.png)
+
+The screenshot shows how in the menu the user can open, quit, or change the user's "status" in a messaging app. (The menu shown has several levels, which is not in scope of the API, but does show the usefulness of providing different options to the end user).
+
+![Telegram app on Windows showing the number of unread messages and a menu](telegramtray.png)
+
+The previous screenshot shows a different instant messaging app that displays a badge with the number of unread messages and a menu that allows you to disable notifications for itself.  
+
+### Collaboration Suites
+A user can directly access their calendar or schedule a meeting right from the app's system status icon. Notice as well in this case the actions offered in the menu are different from the ones specified in the icon in the dock. 
+
+![The Teams app on macOS providing a quick way to schedule a meeting](teamsMacSysTray.png)
+
+### Multimedia Content control
+
+On macOS the VLC media player adds an icon to the system status area to put playback controls and access to other functionality like subtitles for media content that is playing. In general, an app can expose any functionality that it deems useful for the end user.
+
+## Proposed Solution
+
+The scope of this feature is to allow an installed Web app to have an icon on the system's status area if the application is open and running.
+
+The feature consists of several parts, namely, a way for the app to indicate that it wants to have a icon on the status area and a way to specify menu options for when the user right-clicks on the icon. 
+
+### Opting into the System Status icon
+
+For an app to opt into having an icon on the platform's system status area, it can include an icon in the `icons` of the manifest file with a `purpose` of `system_status`.
+
+```json
+{
+  "short_name": "webapp",
+  "name": "Awesome Web App",
+  ...,
+  "icons": [
+    ...,
+    {
+      "src": "icon/statusIcon.png",
+      "purpose" : "status_icon"
+    }
+  ]
+}
+```
+
+This icon will be present ONLY if the application is open and running. This icon can also be different from the one used in the platforms app launcher. Interacting with the icon will open/focus the app's window. Secondary interaction (usually right-clicking) on the icon will bring up the app's system status menu.
+
+### System Status Menu
+As stated before, the icon on the system status for an installed web app will open/focus the app's window when clicked and will have a menu when the user uses the secondary click action. This menu can be specified in the manifest file, and uses the same format as the one for shortcuts. To define it, you add the `secondary_shortcuts` field in the manifest file. 
```

Comment:
I know I previously suggested `secondary_shortcuts`, but if we're trying to formalize this as the "status_icon", then maybe these should be `status_shortcuts`...

Reply:
renamed